### ISSUE TITLE
Improve DX when using hook_default_message_type_category()

### ISSUE THREAD

**Issue by RoSk0** [Gizra/message_notify#9]
When someone wants to use this hook they forced to create message_text field instance  for the category being created.
This could be avoided by implementing hook_ENTITY_TYPE_insert().
I am willing to create a PR is this feature is interesting to maintainers.

**Comment by amitaibu** [comment]
We haven't had people asking for it in the past, so I think a pr isn't
needed in this case.  Thanks :)

**Comment by RoSk0** [comment]
Hm, I would say that you are wrong. There is at least two issues in the issue queue on DO about missing message_text field on new message types with custom message category… What I want to say is that people may not know what to ask for.

**Comment by amitaibu** [comment]
> What I want to say is that people may not know what to ask for.

Fair enough. If you think a PR or a doc page can help with the confusion, any contribution would be appreciated.
